Shocksupershaman - Copy/src/main/java/com/example/shocksupershaman/shocktask1/controller/UserController.java

```diff
@@ -28,21 +28,14 @@ public ResponseEntity<List<UserDTO>> getAllUsers() {
 
     @GetMapping("{id}")
     public ResponseEntity<UserDTO> getUserById(@PathVariable String id) {
-        try {
-            UserDTO user = userService.getUserById(id);
-            return ResponseEntity.ok(user);
-        } catch (UserNotFoundException e) {
-            throw new UserNotFoundException();
-        }
+        UserDTO user = userService.getUserById(id);
+        return ResponseEntity.ok(user);
     }
 
     @PostMapping
-    public ResponseEntity<String> createUser(@RequestBody User user) {
+    public ResponseEntity<User> createUser(@RequestBody @Validated User user) {
         User createdUser = userService.createUser(user);
-        if (createdUser == null) {
-            return ResponseEntity.badRequest().body("Something go wrong");
-        }
-        return ResponseEntity.ok("User created successfully");
+        return ResponseEntity.ok(createdUser);

```

Comment:
Im most cases returning id in response is enough. Return all the fields if there is a requirement only